.github/workflows/tests.yml

```
@@ -1,26 +1,23 @@
-name: Rust
+name: Tests
 
 on:
   push:
     branches: [ "main" ]
+    paths:
+      - '**/Cargo.*'
+      - '**.rs'
+      - '**.config.yml'
```

Oops, looks like I typo'd this path in the original issue description, I meant to write `**/config.yml`.

My intent was to ensure that `axum_example` tests will run when the config file (https://github.com/lazulit3/sea_skipper/blob/main/examples/axum_example/config.yml) has changed (since it could affect test results).
```suggestion
      - '**/config.yml'
```